examples/plot_2D_simulation_example.py

```diff
@@ -63,8 +63,8 @@
 from hidimstat.clustered_inference import clustered_inference
 from hidimstat.desparsified_lasso import desparsified_lasso
 from hidimstat.ensemble_clustered_inference import ensemble_clustered_inference
-from hidimstat.scenario import multivariate_simulation
-from hidimstat.stat_tools import pval_from_cb, zscore_from_pval
+from hidimstat._utils.scenario import multivariate_simulation
+from hidimstat.statistical_tools.stat_tools import pval_from_cb, zscore_from_pval
```

Comment:
This is weird. I guess it should be something like
```
from hidimstat.statistical_tools._utils import pval_from_cb, zscore_from_pval
```

Reply:
`utils` to ackoledge that this is a public module ?

Reply:
done